runtime/queries/metricsview_aggregation_test.go

```diff
@@ -4807,6 +4813,59 @@ func testMetricsViewsAggregation_comparison_with_offset(t *testing.T, rt *runtim
 	require.Equal(t, "news.google.com,3.59,3.69", fieldsToString2digits(rows[i], "dom", "m1", "m1__p"))
 }
 
+func testMetricsViewsAggregation_comparison_iso_timerange(t *testing.T, rt *runtime.Runtime, instanceID string) {
```

Comment:
See the other note about testing using a new file in `runtime/resolvers/testdata` – it will make it easier to do many tests for different values/OLAPs/security policies/etc.

Reply:
Moved this to `runtime/resolvers/testdata`